python/ray/tests/runtime_env_container/test_serve_telemetry.py

```diff
@@ -98,23 +98,30 @@ def check_telemetry_deployment():
     {
         "name": "app1",
         "import_path": "serve_application:app",
+        "route_prefix": "/app1",
         "runtime_env": runtime_env,
     },
 )
 client.deploy_apps(ServeDeploySchema.parse_obj(config))
 wait_for_condition(check_app, app_name="app1", expected="helloworldalice", timeout=300)
 wait_for_condition(check_telemetry_app)
 
-# Deploy with container runtime env set at application level
+deployment_runtime_env = dict(runtime_env)
+deployment_runtime_env["working_dir"] = None
+# Deploy with container runtime env set at deployment level
 config["applications"].append(
     {
         "name": "app2",
-        "import_path": "serve_application:app",
+        "import_path": "read_file:app",
+        "route_prefix": "/app2",
+        "runtime_env": {
```

Comment:
is it related to this PR?

Reply:
no, it was unintentionally being skipped before so some bugs in the test went unnoticed, I fixed them here.